static/src/javascripts/projects/common/modules/commercial/creatives/foundation-funded-logo.js

```diff
@@ -18,22 +20,19 @@ define([
      * * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10022127
      * * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10026327
      */
-    var Template = function ($adSlot, params) {
-        this.$adSlot       = $adSlot;
+    var Template = function (adSlot, params) {
+        this.adSlot        = adSlot instanceof HTMLElement ? adSlot : adSlot[0];
```

Comment:
This will always end up being passed a Bonzo adslot - maybe we should just formalize one type of input or another?